rotkehlchen/api/rest.py

```diff
@@ -3383,12 +3383,15 @@ def _get_current_assets_price(
                     oracle = CurrentPriceOracle.MANUALCURRENT if nft_price_data['manually_input'] is True else CurrentPriceOracle.BLOCKCHAIN  # noqa: E501
                     assets_price[asset] = [Price(nft_price_data['usd_price']), oracle.value]
                 else:
-                    price, oracle = Inquirer().find_price_and_oracle(
+                    price, oracle, used_main_currency = Inquirer().find_price_and_oracle_and_check_main_currency(  # noqa: E501
                         from_asset=asset,
                         to_asset=target_asset,
                         ignore_cache=ignore_cache,
                     )
-                    assets_price[asset] = [price, oracle.value]
+                    if used_main_currency is True:
+                        assets_price[asset] = [price, oracle.value, used_main_currency]
+                    else:
+                        assets_price[asset] = [price, oracle.value]
```

Comment:
```suggestion
                    assets_price[asset] = [price, oracle.value, used_main_currency]
```

Why not just forward in any case? Why do you need this extra if?

Reply:
Yeah, originally I actually made it having all three arguments. But then read what kelsos wrote in the issue and saw:

> Backend checks if the latest manual price is the main currency and then returns a flag 1 in the tuple, and returns the price unconverted, and the flag as the third tuple value

From `and then returns a flag 1` + `the flag as the third tuple value` I thought that the flag should exist only if this special case happened. But I might understood it wrong though. (will ask kelsos to clarify)